examples/analytics_specs/provider.tf

```diff
@@ -0,0 +1,5 @@
+provider "mongodbatlas" {
+  base_url    = "https://cloud-dev.mongodb.com/"
```

Comment:
would remove this base url configuration as end users will not be configuring this in regular case. Applies to all examples.